lib/Common/Memory/PageAllocator.cpp

```diff
@@ -1387,6 +1394,37 @@ PageAllocatorBase<T>::AddFreePageCount(uint pageCount)
 
 template<typename T>
 void
+PageAllocatorBase<T>::PartialDecommitPages(__in void * address, size_t pageCountTotal, __in void* decommitAddress, size_t pageCountToDecommit, __in void * segmentParam)
+{
+    // TODO: use a specialized PageHeapPageAllocator to simplify the page allocating logic for pageheap
+    if (pageCountTotal > this->maxAllocPageCount) 
+    {
+        SegmentBase<T> * segment = (SegmentBase<T>*) segmentParam;
```

Comment:
should be. let me try

---
In reply to: [60988863](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60988863) [](ancestors = 60988863)